src/main/java/mServer/crawler/sender/arte/ArteJsonObjectToDatenFilmCallable.java

```diff
@@ -124,33 +109,33 @@ public DatenFilm call() {
         }
         return film;
     }
-    
-    
-    private LocalDateTime readDateTime(String aFilmDetails) throws IOException
-    {
-        DateTimeFormatter dateTimeFormatter = new DateTimeFormatterBuilder().appendPattern("EEEE', 'd'. 'MMMM' um 'H.mm' Uhr'")
-            .parseDefaulting(ChronoField.HOUR_OF_DAY, 0)
-            .parseDefaulting(ChronoField.MINUTE_OF_HOUR, 0)
-            .parseDefaulting(ChronoField.SECOND_OF_MINUTE, 0)
-            .parseDefaulting(ChronoField.YEAR, 2017)
-            .toFormatter(Locale.GERMANY); 
-            Matcher matcher = Pattern.compile(REGEX_PATTERN_DATETIME).matcher(aFilmDetails);
-        if(matcher.find())
+
+    private DatenFilm createFilm(String thema, String urlWeb, String titel, ArteVideoDTO video, String date, String time, LocalTime durationAsTime, String beschreibung) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Method has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00107)